trestle/core/generators.py

```diff
@@ -58,19 +58,20 @@ def generate_sample_value_by_type(
         return False
     if type_ is int:
         return 0
-    if type_ is str:
-        if field_name == 'oscal_version':
-            return OSCAL_VERSION
-        return 'REPLACE_ME'
     if type_ is float:
         return 0.00
     if safe_is_sub(type_, ConstrainedStr) or (hasattr(type_, '__name__') and 'ConstrainedStr' in type_.__name__):
         # This code here is messy. we need to meet a set of constraints. If we do
         # TODO: handle regex directly
         if 'uuid' == field_name:
             return str(uuid.uuid4())
+        # some things like location_uuid in lists arrive here with field_name=''
+        if type_.regex and type_.regex.pattern.startswith('^[0-9A-Fa-f]{8}'):
+            return const.SAMPLE_UUID_STR
         if field_name == 'date_authorized':
             return str(date.today().isoformat())
+        if field_name == 'date_datatype':
+            return '2023-01-01'
```

Comment:
Not sure where this comes from, would it be better to keep in isoformat as well?

Reply:
I'm not sure why I needed to add this.  It doesn't seem needed now - it may have been a side effect of an early attempt to remove root.  I have removed it.